chainweb.cabal

```diff
@@ -539,32 +538,22 @@ executable chainweb-miner
           chainweb
 
         -- external
-        , aeson >= 1.4
-        , async >= 2.2
         , base >= 4.11 && < 5
-        , base-prelude >= 1.3
-        , base16-bytestring >= 0.1
-        , bytes >= 0.15
-        , bytestring >= 0.10
         , connection >= 0.2
-        , containers >= 0.5
         , errors >= 2.3
+        , generic-lens >= 1.2
         , http-client >= 0.5
         , http-client-tls >= 0.3
         , mwc-random >= 0.13
         , optparse-applicative >= 0.14
-        , pretty-simple >= 2.1
         , retry >= 0.7
+        , rio >= 0.1.9
```

Comment:
RIO offered the simplest logging infrastructure that I knew how to get running quickly. Its logging has nice looking, reasonable defaults. It cleans up a lot of code below, too.